firmware/config/boards/prometheus/prometheus-common.mk

```diff
@@ -22,12 +22,10 @@ ifeq ($(DEBUG_LEVEL_OPT),)
 endif
 
 ifeq ($(PROMETHEUS_BOARD),405)
-	LDSCRIPT = $(PROJECT_DIR)/config/boards/prometheus/STM32F405xG.ld
-	DDEFS += -DDEFAULT_ENGINE_TYPE=PROMETHEUS_DEFAULTS -DSTM32F405xx -DFIRMWARE_ID=\"prometeus405\"
+	DDEFS += -DDEFAULT_ENGINE_TYPE=PROMETHEUS_DEFAULTS -DFIRMWARE_ID=\"prometeus405\"
 else
 	# Override DEFAULT_ENGINE_TYPE
-	LDSCRIPT = $(PROJECT_DIR)/config/boards/prometheus/STM32F469xI.ld
-	DDEFS += -DDEFAULT_ENGINE_TYPE=PROMETHEUS_DEFAULTS -DSTM32F469xx -DFIRMWARE_ID=\"prometeus469\"
```

Comment:
If you remove 469 define from here, then where is it now? We cannot use 405 defs for 427 and 469, these are different chips with their own ChibiOS/os/common/ext/ST/STM32F4xx/* includes and stm32f4xx_hal_flash_ex.* support.

Reply:
> We cannot use 405 defs for 427 and 469, these are different chips
This isn't true.  The F42x and F469 are strict supersets of what's supported on the F40x.  Just compare stm32f405xx.h with stm32f69xx.h - Save for the addition of features like DMA2D, camera interface, LCD controller, etc. they're the same file.  ST even claims this in their own migration guide that it's 100% software compatible to go from F405/7 to F42x and F469.

Reply:
And you can't even use the slightly faster 180MHz core clock on the 42x and 469 since that breaks USB.  The core clock has to be an integer multiple of 48mhz, which 168mhz is, and 180mhz is not.